.gitignore

```diff
@@ -6,3 +6,5 @@ dependency-reduced-pom.xml
 .classpath
 .project
 .settings/
+.DS_Store
+gradoop-flink/src/test/resources/data/csv/input_indexed/
```

Comment:
this is where the currently failing test gets its data from, i.e. no need to ignore it

Reply:
.DS_Store alway auto create by MacOSX